crates/services/txpool/src/containers/dependency.rs

```diff
@@ -663,6 +652,11 @@ impl Dependency {
             }
         }
 
+        if let PoolTransaction::Blob(checked_tx, _) = tx.as_ref() {
```

Comment:
This doesn't exactly match the name of the function. Maybe we can rename it `remove_all_dependencies`?

Reply:
This function calls `recursively_remove_all_dependencies`, so the name is right=)

Reply:
Yeah. On a second look I think it's fine.